### PR TITLE
Home page banner/landing section #310

### PR DESCRIPTION
**Pull request by taoliu12**
issue [#310](https://github.com/Nonprofit-Exchange-Hub/web-app/issues/310) 

Done, as I showed during our team meeting.

### CODE REVIEW

**Comment by taoliu12** [comment]
Still need to fix eslint errors. Holding off on merging until I fix the eslint errors.

**Comment by taoliu12** [comment]
fixed eslint errors. Should be ready to merge now.

**Review by esteban-gs — CHANGES_REQUESTED** [review]
See comments

**Comment by taoliu12** [comment]
@esteban-gs Refactored as much as it currently make sense to me. 
Should be ready to merge, but also let me know if you need anything else.

File: client/src/components/BannerSection.tsx [hunk 1]
```diff
@@ -1,94 +1,72 @@
...
+function BannerSection() {
+  const classes = useStyles();
+
+  return (
+    <Box>
+      <Grid container spacing={3} sx={{ height: '600px' }}>
+        <Grid xs={6} item justifyContent="center" sx={{ display: 'flex' }}>
+          <Box sx={{ width: '550px', mt: '180px', mb: '66px', ml: '40px' }}>
+            <BannerText />
+            <CTAButton text="Join Now" />
```
Comment (esteban-gs) [inline client/src/components/BannerSection.tsx:53]: ✅

**Review by esteban-gs — APPROVED** [review]
Looks good to me! 🚀

**Pull request merged**